src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -190,7 +190,6 @@ private void LaunchDynamo(bool isCheckboxChecked)
             DynamoModel.RequestUpdateLoadBarStatus -= DynamoModel_RequestUpdateLoadBarStatus;
             StaticSplashScreenReady -= OnStaticScreenReady;
             Close();
-            Application.Current.MainWindow = dynamoView;
```

Comment:
It seems we did this later in DynamoView code but probably indeed not needed here